common/src/main/java/org/valkyrienskies/mod/mixin/mod_compat/cc_tweaked/MixinTurtleMoveCommand.java

```diff
@@ -1,57 +1,48 @@
-package org.valkyrienskies.mod.forge.mixin.compat.cc_tweaked;
+package org.valkyrienskies.mod.mixin.mod_compat.cc_tweaked;
 
+import com.google.common.collect.Streams;
 import dan200.computercraft.api.turtle.TurtleCommandResult;
 import dan200.computercraft.shared.turtle.core.TurtleMoveCommand;
 import dan200.computercraft.shared.turtle.core.TurtlePlayer;
 import net.minecraft.core.BlockPos;
 import net.minecraft.server.level.ServerLevel;
 import net.minecraft.world.level.ChunkPos;
-import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.state.BlockState;
 import org.joml.Vector3d;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.Pseudo;
-import org.spongepowered.asm.mixin.Unique;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
 import org.valkyrienskies.core.api.ships.Ship;
-import org.valkyrienskies.mod.common.VSGameUtilsKt;
-
-import java.util.List;
+import org.valkyrienskies.mod.api.ValkyrienSkies;
 
 @Pseudo
 @Mixin(TurtleMoveCommand.class)
 public abstract class MixinTurtleMoveCommand {
     @Inject(method = "canEnter", at = @At("RETURN"), remap = false, cancellable = true)
     private static void ValkyrienSkies2$canEnter(
-            final TurtlePlayer turtlePlayer, final ServerLevel world, final BlockPos position, final CallbackInfoReturnable<TurtleCommandResult> cir) {
+        TurtlePlayer turtlePlayer, ServerLevel world, BlockPos position,
+        CallbackInfoReturnable<TurtleCommandResult> cir) {
         if (cir.getReturnValue().isSuccess()) {
-            final Ship ship = VSGameUtilsKt.getShipManagingPos(world, position);
+            final Ship ship = ValkyrienSkies.getShipManagingBlock(world, position);
             if (ship == null) {
-                final Ship iShip = VSGameUtilsKt.getShipManagingPos(world, getShipPosFromWorldPos(world, position));
-                if (iShip != null) {
-                    cir.setReturnValue(TurtleCommandResult.failure("ship"));
+                final boolean notInAir = Streams
+                    .stream(ValkyrienSkies.positionToNearbyShipsAndWorld(world, position.getX() + 0.5, position.getY() + 0.5, position.getZ() + 0.5, 0.1))
+                    .map(pos -> ValkyrienSkies.getShipManagingBlock(world, pos))
+                    .map(s -> ValkyrienSkies.positionToShip(s, new Vector3d(position.getX() + 0.5, position.getY() + 0.5, position.getZ() + 0.5)))
+                    .map(pos -> world.getBlockState(BlockPos.containing(ValkyrienSkies.toMinecraft(pos))))
+                    .anyMatch(BlockState::isAir);
```

Comment:
Please follow #1051

Reply:
Ended up re-implementing the stream and the check another way as explained in that PR